docs/usage/integration.md

```diff
@@ -0,0 +1,413 @@
+# Integration
+
+TigerBeetle is designed to guard against bugs not only in its [own code](../TIGER_STYLE.md), but
+at the boundaries, in the application code which interfaces with TigerBeetle.
+This is exhibited by the client's API design, which may be [surprising](#retries) when contrasted
+with a more conventional database.
+
+Strict consistency guarantees at the database level simplify application logic and error handling
+farther up the stack.
+
+
+
+## Consistency
+
+TigerBeetle provides strict serializability (serializability + linearizability) to each
+[client session](#client-sessions).
+
+But consistency models are notoriously arcane — what specifically does this mean for applications
+in practice?
+
+- [**Client sessions**](#client-sessions)
+  - A client session may have at most one in-flight request.
```

Comment:
I don't want to get you bogged down in the weeds. But is `in-flight` a generally understood idea?

Reply:
:thinking: I thought it was, but I don't know for sure. Do you think "pending" would be more clear?

Reply:
Is `may have at most one request at a time` equivalent?

Reply:
I think that is a little less accurate, clients do have multiple requests, they just queue them up locally until the in-flight/pending one finishes.

Reply:
ok!